frontend/uddhar-frontend/src/coordinator/components/TableWithPagination.jsx

```
@@ -0,0 +1,199 @@
+              &laquo;
+            </button>
+            <button
+              onClick={() => handleClick(currentPage - 1)}
+              disabled={currentPage === 1}
+              className="px-2 py-1 border rounded cursor-pointer"
+            >
+              &lsaquo;
+            </button>
+            {[...Array(totalPages)].map((_, index, counter =0) => (
+              <button
+                key={counter++}
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)